src/components/calcite-pick-list-group/calcite-pick-list-group.tsx

```diff
@@ -21,17 +21,42 @@ export class CalcitePickListGroup {
    */
   @Prop({ reflect: true }) textGroupTitle: string;
 
+  // --------------------------------------------------------------------------
+  //
+  //  Private Properties
+  //
+  // --------------------------------------------------------------------------
+
+  @Element() el: HTMLElement;
+
+  @State() hasParent: boolean;
+
+  // --------------------------------------------------------------------------
+  //
+  //  Lifecycle
+  //
+  // --------------------------------------------------------------------------
+
+  componentWillLoad() {
+    this.hasParent = this.el.querySelector("[slot=parentItem]") !== null;
+  }
+
   // --------------------------------------------------------------------------
   //
   //  Render Methods
   //
   // --------------------------------------------------------------------------
 
   render() {
+    const classes = {
+      [CSS.container]: true,
+      [CSS.indent]: this.hasParent
```

Comment:
Maybe this should be:

```ts
[CSS.indent]: this.el.querySelector("[slot=parentItem]") !== null
```

Because if a slot is added at after component initialization this won't work.

Reply:
you mean ditch the state. Yea seems ok

Reply:
@pr3tori4n this still needs addressing.

Reply:
@pr3tori4n I have a branch with my style changes and some TSX updates too. I can add this to my branch...also have added a SLOTS const in resources with `parentItem`.